From ecbf509871f16438d69ae157c690b7d9bedd62f0 Mon Sep 17 00:00:00 2001
From: Rudi Heitbaum <rudi@heitbaum.com>
Date: Sun, 27 Feb 2022 03:07:58 +0000
Subject: [PATCH] AX_CC_MAXOPT: Fix nvhpc case

Missing ;; in the ax_cv_c_compiler_vendor case statement causing
syntax error with libffi configure:
  line y: syntax error near unexpected token `)'
  line y: `    gnu)'

Signed-off-by: Rudi Heitbaum <rudi@heitbaum.com>
---
 m4/ax_cc_maxopt.m4 | 1 +
 1 file changed, 1 insertion(+)

diff --git a/m4/ax_cc_maxopt.m4 b/m4/ax_cc_maxopt.m4
index 05218e3..58e267b 100644
--- a/m4/ax_cc_maxopt.m4
+++ b/m4/ax_cc_maxopt.m4
@@ -146,6 +146,7 @@ if test "x$ac_test_CFLAGS" = "x"; then
     nvhpc)
      # default optimization flags for nvhpc
      CFLAGS="$CFLAGS -O3"
+     ;;
 
     gnu)
      # default optimization flags for gcc on all systems
-- 
2.25.1

